go/obscuronode/host/host.go

```diff
@@ -140,8 +118,8 @@ func NewObscuroAggregator(
 		ethWallet:       ethWallet,
 	}
 
-	if cfg.HasRPC {
-		host.clientServer = NewClientServer(*cfg.RPCAddress, &host)
+	if config.HasClientRPC {
+		host.clientServer = NewClientServer(config.ClientRPCAddress, &host)
 	}
 
```

Comment:
At my last place there was loads of config and whenever a process started it dumped the config to the log, was super helpful when you were debugging an issue to be able to see the full config in one place (useful for making sure it had been run properly for example).

Worth dumping the config out to the log here or in `host.Start()` potentially?

Reply:
Yeah, but I'll do that as a follow-up.

Reply:
Added a todo.